Mathlib/Data/Real/Basic.lean

```diff
@@ -639,3 +639,8 @@ end Real
   `f (r + s) ≤ max (f r) (f s)` for all `r s : R`. -/
 def IsNonarchimedean {A : Type _} [Add A] (f : A → ℝ) : Prop :=
   ∀ r s, f (r + s) ≤ max (f r) (f s)
+
+/-- A function `f : R → ℝ` is power-multiplicative if for all `r ∈ R` and all positive `n ∈ ℕ`,
+`f (r ^ n) = (f r) ^ n`. -/
+def IsPowMul {R : Type _} [Semiring R] (f : R → ℝ) :=
+  ∀ (a : R) {n : ℕ} (_ : 1 ≤ n), f (a ^ n) = f a ^ n
```

Comment:
```suggestion
  ∀ (a : R) {n : ℕ}, 1 ≤ n → f (a ^ n) = f a ^ n
```

Reply:
Why asking for a semiring here?

Reply:
I was always applying this to rings, but only `[Pow R ℕ]` is required here, so I have changed the hypothesis.